src/styles/colors.module.scss

```diff
@@ -1,2 +1,3 @@
 $bg-1: #f4fdfd;
 $bg-2: #fdffff;
+$primary-3: #08566b;
```

Comment:
We have a `primary-3`
https://github.com/hack-the-6ix/react-ui/blob/2022/src/styles/_colors.module.scss#L15